src/libs/actions/Report.js

```diff
@@ -691,11 +690,10 @@ function fetchOrCreateChatReport(participants, shouldNavigate = true) {
  * Get the actions of a report
  *
  * @param {Number} reportID
- * @param {Number} [offset]
  * @returns {Promise}
  */
-function fetchActions(reportID, offset) {
-    const reportActionsOffset = !_.isUndefined(offset) ? offset : -1;
+function fetchActions(reportID) {
+    const reportActionsOffset = -1;
```

Comment:
I dunno what is the purpose of the log below this line, but doubt we need it now that `reportActionsOffset` is hardcoded.